build.fsx

```diff
@@ -611,17 +617,17 @@ open Fake.Core.TargetOperators
 "BuildTests"
   ==> "DotnetTests"
 
-"BuildTests"
-  ==> "RunTests"
+//"BuildTests"
+  //==> "RunTests"
```

Comment:
Are the test running on appveyor/travis?

Reply:
Yes but they now use dotnet test instead of nunit runner

Reply:
Aaa, so that's why 😄

Reply:
I remove it in the next PRs